core/src/migrations/000063-addSyncModeToDestination.ts

```diff
@@ -0,0 +1,64 @@
+export default {
+  up: async function (migration, DataTypes) {
+    await migration.sequelize.transaction(async () => {
+      await migration.addColumn("destinations", "syncMode", {
+        type: DataTypes.STRING(191),
+        allowNull: true,
+      });
+
+      // Pipedrive
+      await migration.sequelize.query(
+        `UPDATE "destinations" SET "syncMode"='sync' WHERE "state"='ready' AND "type"='pipedrive-export' AND "locked" IS NULL`
+      );
+
+      // Intercom
+      const [intercomOptions] = await migration.sequelize.query(
+        `SELECT "ownerId" AS "id", "key", "value" FROM "options" o JOIN "destinations" d ON o."ownerId"=d.id WHERE "state"='ready' AND d.type='intercom-export-contacts' AND "locked" IS NULL`
+      );
+
+      const intercomDestinations = intercomOptions.reduce((dest, opt) => {
+        if (!dest[opt.id]) {
+          dest[opt.id] = {};
+        }
+        dest[opt.id][opt.key] = opt.value;
+        return dest;
+      }, {});
+
+      for (const destId in intercomDestinations) {
+        const dest = intercomDestinations[destId];
+        let syncMode;
+
+        if (dest.creationMode !== "Enrich" && dest.removalMode !== "Skip") {
```

Comment:
Should these be lower case?

Reply:
Nope! These are intercom's options, which were upper case